doc-content/enterprise-only/integration/entando-integrating-proc.adoc

```diff
@@ -1,34 +1,11 @@
 [id='entando-integrating-proc_{context}']
 
-= Configuring {KIE_SERVERS} in Entando App Builder
+= Configuring {PRODUCT} with Entando App Builder
```

Comment:
Since this is no longer a procedure, I'd call it "{PRODUCT} configuration with Entando App Builder".

You'll need to also change the anchor ID and file name to say "-con" instead of "-proc" BUT, first search the entire kie-docs project for both current names to find if and where they're currently used and update the refs and includes to the new name, with "-con". Otherwise it will break. This is like what we discussed the other day in our single-source convo.

Reply:
Done